app/assets/stylesheets/master.scss

```diff
@@ -0,0 +1,23 @@
+@import "bootstrap";
+
+//------start chess board------//
+.board {
+  width:auto;
+  border: 2px solid black;
+  margin: auto;
+  box-shadow: 5px 5px 6px 2px black;
+}
+.board, td {
+  border: 1px solid black;
+}
+td {
+  width: 60px;
+  height: 60px;
+}
```

Comment:
I have tried out your app. It looks really awesome. The only feedback I have is maybe using a percentage here instead of hardcoded values to make the board more responsive with different sizes.